src/app/core/nav-menu/components/license/license.component.ts

```diff
@@ -0,0 +1,34 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+
+@Component({
+  selector: 'cs-license',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
```

Comment:
Are there any rules regarding inline templates and styles? How do you decide to make this component with inline template and styles instead of putting them into separate files?

Reply:
No, we don't have any rule about this. I make this based on component size and simplicity.